perceptual_filter/Feature[] features
perceptual_filter/Label[] labels
bool new_sequence
uint64 sequence_number
string sequence_boundary
uint64 timeID